docs/ops/pooling/MaxPool_8.md

```diff
@@ -0,0 +1,368 @@
+## MaxPool <a name="MaxPool"></a> {#openvino_docs_ops_pooling_MaxPool_8}
+
+**Versioned name**: *MaxPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Performs the max pooling operation on input.
+
+**Detailed description**: Input shape can be either 3D, 4D, or 5D. The max pooling operation is performed with respect to input shape from the third dimension to the last dimension. If paddings are used, during the pooling calculation their values are `-inf`. The max pooling operation involves sliding a filter over each channel of a feature map and downsampling by choosing the largest value within the region covered by the filter. [Article about max pooling in Convolutional Networks](https://deeplizard.com/learn/video/ZjM_XQa5s6s).
+
+**Attributes**: *Pooling* attributes are specified in the `data` node, which is a child of the layer node.
+
+* *strides*
+
+  * **Description**: *strides* is a distance (in pixels) to slide the window on the feature map over the (z, y, x) axes for 3D poolings and (y, x) axes for 2D poolings. For example, *strides* equal to "4,2,1" means sliding the window 4 pixels at a time over depth dimension, 2 over height dimension, and 1 over width dimension.
+  * **Range of values**: integer values starting from 0
+  * **Type**: int[]
+  * **Required**: *yes*
+
+* *dilations*
+
+    * **Description**: *dilations* specify the index of the next pixel to select when pooling. If not present, the dilation defaults to 1, meaning the adjacent pixel is chosen. A value of 2 indicates that one pixel is skipped and every other pixel is considered. Dilations specify one value for each spatial axis of the kernel: `(z, y, x)` for 3D poolings and `(y, x)`  for 2D poolings.
+    * **Range of values**: integer values starting from 0
+    * **Type**: int[]
+    * **Default value**: `[1,1,...]`
+    * **Required**: *no*
+
+* *pads_value*
```

Comment:
Suggestion to add a requirement that the value needs to fit in the data type of the operator's input. Otherwise the op might produce undefined behavior.